xds/src/main/java/io/grpc/xds/XdsEndpointResource.java

```diff
@@ -49,8 +49,9 @@
 class XdsEndpointResource extends XdsResourceType<EdsUpdate> {
   static final String ADS_TYPE_URL_EDS =
       "type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment";
-  static final String GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS =
-      "grpc.experimental.xdsDualstackEndpoints";
+
+  public static final String GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS =
+      "GRPC_EXPERIMENTAL_HAPPY_EYEBALLS";
```

Comment:
HAPPY_EYEBALLS? This should be `"GRPC_EXPERIMENTAL_XDS_DUALSTACK_ENDPOINTS"`, per the gRFC, right?

Reply:
Fixed